src/main/target/OMNIBUSF4FW/target.h

```diff
@@ -193,8 +192,13 @@
 #define RSSI_ADC_PIN            PA0  // Direct from RSSI pad
 
 // Allegro Systems ACS781KLRTR-150U-T
+#if defined(OMNIBUSF4V6)
+#undef CURRENT_METER_SCALE_DEFAULT
```

Comment:
This isn't disabling the current meter, this is just setting the scaling to default.

Reply:
Yes, sorry, my intention is to leave it by default, do not remove

Reply:
In this case you don't have to `undef` them - they will be set to default if not defined.

Reply:
I know,but the problem is that OMNIBUSF4FW (Fireworks) needs the definition

Reply:
@mikeller the solution will be defined only in OMNIBUSF4FW? but this is the base target

Reply:
or this would be correct?

#if defined(OMNIBUSF4V6)
#else
#define CURRENT_METER_SCALE_DEFAULT  176
#define CURRENT_METER_OFFSET_DEFAULT -18500
#endif

Reply:
Yes.

```
#if !defined(OMNIBUSF4V6)
#define CURRENT_METER_SCALE_DEFAULT 176
#define CURRENT_METER_OFFSET_DEFAULT -18500
#endif
```

is even better.

Reply:
Thanks @mikeller ,I've corrected :)